docs/man/encrypt/_index.md

```diff
@@ -10,6 +10,15 @@ command:
     - name: attr
       shorthand: a
       description: Attribute value Fully Qualified Names (FQNs, i.e. 'https://example.com/attr/attr1/value/value1') to apply to the encrypted data.
+    - name: wrapping-key-algorithm
+      shorthand: w
```

Comment:
I think we should remove this shorthand because it's `-s` for `session-key-algorithm` on decrypt and `-w` for `wrapping-key-algorithm` on encrypt, but both have the same algorithm values which can be confusing. No shorthand for either one would be preferable.